libse/Forms/FixCommonErrors/FixShortLinesAll.cs

```diff
@@ -9,13 +9,22 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
             var language = Configuration.Settings.Language.FixCommonErrors;
             string fixAction = language.MergeShortLineAll;
             int noOfShortLines = 0;
+            bool isCRLF = Environment.NewLine.Length == 2;
             for (int i = 0; i < subtitle.Paragraphs.Count; i++)
             {
                 Paragraph p = subtitle.Paragraphs[i];
                 if (callbacks.AllowFix(p, fixAction))
                 {
                     string s = HtmlUtil.RemoveHtmlTags(p.Text, true);
-                    if (s.Replace(Environment.NewLine, " ").Replace("  ", " ").Length < Configuration.Settings.Tools.MergeLinesShorterThan && p.Text.Contains(Environment.NewLine))
+                    int newLineCharCount = Utilities.CountTagInText(s, Environment.NewLine);
+                    int len = s.Length;
+                    if (isCRLF)
+                    {
+                        // keep one character to represent space
+                        // e.g foobar\r\nfoobar => foobar foobar
```

Comment:
this is the text **foobar foobar** we want to compare its length again **Configuration.Settings.Tools.MergeLinesShorterThan**

Updated :)